tools/scripts/build-c-shared-linux.sh

```
@@ -28,7 +28,7 @@ echo "Removing existing .so and .h files..."
 rm -f build/libdefradb.so build/libdefradb.h
 
 echo "Building shared object..."
-CGO_ENABLED=1 GOARCH=amd64 GOOS=linux go build -tags cshared $BUILD_FLAGS \
+CGO_ENABLED=1 GOARCH=amd64 GOOS=linux go build -tags "cshared ${BUILD_TAGS:-}" $BUILD_FLAGS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)